### DISCUSSION TITLE
How to get it working?

### DISCUSSION THREAD

**Discussion by maxkes** [russhughes/st7789_mpy#80] [Q&A]
Hi,
I wanted to use the lib on Pyboard v1.1 and used the precompiled firmware.
But I cannot
`import st7789`
because there is so such module. I have not copied any files on the board. Do I need to copy some files from the repo or should it work just with the precompiled firmware.
Thanks, Max

**Comment by russhughes** [comment]
Something has changed in the build system and it is not finding the st7789 module. So far I have not found a solution.

**Reply by maxkes** [reply]
Thanks for the quick reply.
Do you have an older working Firmware Version? This would be great for me because I would like to usw the Display in a Test Setup.

**Reply by maxkes** [reply]
But on ESP32 it's working fine 👍 
On pyboard I tested
`help('modules')`
and the st7789 is not in the list. On ESP32 it's in the list.

**Comment by russhughes** [comment]
Just pushed an updated firmware for the Pyboard v1.1

**Reply by maxkes** [reply]
I tested it: working !

**Comment by russhughes** [comment]
Excellent!
